common/app/views/fragments/javaScriptLaterSteps.scala.html

```diff
@@ -334,6 +334,57 @@
 
                 });
 
+                @for(membershipAccess <- item.membershipAccess if conf.Switches.MembersAreaSwitch.isSwitchedOn) {
```

Comment:
@gklopper Absolutely agree, any guidance on the best way to add tests for this would be appreciated.